projects/js-packages/publicize-components/src/components/share-post/index.jsx

```diff
@@ -119,6 +136,10 @@ export function SharePostButton() {
 			environment: getSiteType(),
 		} );
 
+		if ( isDirtyPost && isAutosaveablePost && hasMediaFeatures ) {
```

Comment:
I am not sure I understand the purpose of `hasMediaFeatures` here?

Can you please explain? May be added a comment in the code itself?

Reply:
Yeah sorry. It was from p1741168564582269-slack-C02JJ910CNL but I'll add a small summary here

Reply:
Added, let me know if that clears it up

Reply:
Yeah, thanks.